drones/image_processing/normalization.py

```diff
@@ -0,0 +1,250 @@
+import numpy as np
+import cv2 as cv
+
+
+def min_max_norm(image: np.ndarray) -> np.ndarray:
+    """Apply min max norm on given image. It should be used with OpenCv
+
+    Parameters:
+    ----------
+    image : np.ndarray
+        image to be normalized
+
+    Returns:
+    ----------
+    image : np.ndarray
+        normalized version of input image
+    """
+    norm_image = np.zeros((image.shape[0], image.shape[1]))
+    return cv.normalize(image, norm_image, 0, 255, cv.NORM_MINMAX)
+
+
+def zero_one_norm(image: np.ndarray) -> np.ndarray:
+    """Apply zero one pixel range. It should be used with AI
```

Comment:
What does it mean "It should be used with AI"?

Reply:
same here